==== triton/lib/Conversion/TritonToTritonGPU/TritonGPUConversion.cpp#3 - triton/lib/Conversion/TritonToTritonGPU/TritonGPUConversion.cpp#4 ====
--- triton/lib/Conversion/TritonToTritonGPU/TritonGPUConversion.cpp 2024-06-10 07:15:59.000000000 -0700
+++ triton/lib/Conversion/TritonToTritonGPU/TritonGPUConversion.cpp 2024-07-23 06:46:59.000000000 -0700
@@ -57,8 +57,9 @@
   addArgumentMaterialization([&](OpBuilder &builder,
                                  RankedTensorType tensorType, ValueRange inputs,
                                  Location loc) -> std::optional<Value> {
-    llvm_unreachable("Argument rematerialization should not happen in Triton "
-                     "-> TritonGPU conversion");
+    // TODO(b/354860562): reenable or remove.
+    // llvm_unreachable("Argument rematerialization should not happen in Triton "
+    //                 "-> TritonGPU conversion");
     return std::nullopt;
   });